Makefile

```diff
@@ -1,6 +1,6 @@
 
 setup_minikube:
```

Comment:
This whole rule seems dangerous/odd to me.

Reply:
It absolutely is. I'll call that out in the README and link to relevant minikube manual setup instructions but the convenience method for those that like to live dangerously is p.nice